src/storage/src/hummock/compactor/shared_buffer_compact.rs

```diff
@@ -92,28 +92,59 @@ async fn compact_shared_buffer(
 ) -> HummockResult<Vec<SstableInfo>> {
     let mut start_user_keys = payload
```

Comment:
We can use a better name. For example, size_and_start_keys.

Reply:
fixed